### PR TITLE
IGVF-940-comments-on-dependencies

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/measurement_set.json [hunk 1]
```diff
@@ -116,6 +116,7 @@
         "moi": {
             "title": "Multiplicity Of Infection",
             "description": "The actual multiplicity of infection (MOI) for vectors introduced to the biosample used to produce data in this measurement set. At least one construct library must be specified in order to specify MOI. This property should capture the actual MOI, and not the targeted MOI.",
+            "comment": "Use of this proeprty requires construct_libraries.",
```
Comment (jenjou) [inline src/igvfd/schemas/measurement_set.json:119]: Minor typo here

File: src/igvfd/schemas/sample.json [hunk 2]
```diff
@@ -38,19 +38,19 @@
...
             ],
-            "comment": "Sample with sorted_fraction must include sorted_fraction_detail."
+            "comment": "A sample with sorted_fraction must include sorted_fraction_detail."
         },
         "sorted_fraction_detail": {
             "required": [
                 "sorted_fraction"
             ],
-            "comment": "Sample with sorted_fraction_deatil must include sorted_fraction."
+            "comment": "A sample with sorted_fraction_deatil must include sorted_fraction."
```
Comment (jenjou) [inline src/igvfd/schemas/sample.json:53]: Looks like an existing typo, might as well fix it now.

File: src/igvfd/schemas/sample.json [hunk 3]
```diff
@@ -109,14 +109,14 @@
...
-            "comment": "The parent sample is expected to have similar properties to its fraction.",
+            "comment": "The parent sample is expected to have similar properties to its fraction. A sample with sorted_fraction must include sorted_fraction_detail.",
             "type": "string",
             "linkTo": "Sample"
         },
         "sorted_fraction_detail": {
             "title": "Sorted Fraction Detail",
             "description": "Detail for sample sorted into fractions capturing information about sorting.",
-            "comment": "For example, FACS 10% expression of CTCF-GFP",
+            "comment": "For example, FACS 10% expression of CTCF-GFP. A sample with sorted_fraction_deatil must include sorted_fraction.",
```
Comment (jenjou) [inline src/igvfd/schemas/sample.json:119]: Minor typo

File: src/igvfd/schemas/treatment.json [hunk 4]
```diff
@@ -181,13 +181,13 @@
                 "μg/mL",
                 "μM"
             ],
-            "comment": "amount_units must be specified if amount is specified"
+            "comment": "Use of this property requires amount. If depletion is false this property is required. "
```
Comment (jenjou) [inline src/igvfd/schemas/treatment.json:184]: Extra white space at the end here

File: src/igvfd/schemas/whole_organism.json [hunk 5]
```diff
@@ -80,6 +80,12 @@
             "linkEnum": [
                 "25d5ad53-15fd-4a44-878a-ece2f7e86509"
             ]
+        },
+        "part_of": {
+            "comment": "Whole organisms should not specify as part of any other sample."
+        },
+        "pooled_from": {
+            "comment": "Whole organisms should not specify as pooled from any other sample."
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/whole_organism.json]: I think the original wording was to indicate that it shouldn't be submitted. By replacing with "be" it sounds more like a suggestion. How about: "Do not submit. Whole organism samples should not be pooled from any other sample."

**Review by jenjou — DISMISSED** [review]

**Pull request merged**